### ISSUE TITLE
Load Network

### ISSUE THREAD

**Issue by Aleoli2** [Artelnics/opennn#113]
The other issue is close, but the load function doesn't work to me.
I have tried to use the constructor of Neural Network with the string of the filename (that calls to load function) and with tinyXML document (to call directly to from_XML) and none have worked. The terminal is closed without exception. If I put a wrong filename, it says that the document cannot be load, so the document is load correctly.

**Comment by GuillermoArtelnics** [comment]
Hi,

Try with declaring the path of the file before constructor : 
'''
string filename = "path_to_folder/filename.xml";
NeuralNetwork neural_network(filename);
'''
Let me know if this works for you, or what mistakes you come up with. If the issue remains, write me the information about you OS, software and compiler.

**Comment by Aleoli2** [comment]
Hi,
The same error it's happening. My OS is Debian 10 and I'm using QT. I compile OpenNN so I'm not using the .pro example. Using the QT debugger I have seeing that the program receives a segmentation fault in inputs_from_XML function.
My compiler is the GNU Compiler version  8.3.0

**Comment by GuillermoArtelnics** [comment]
And you try using .pro?

**Comment by JavierSanchezB** [comment]
I just checked it on Ubuntu 18.04, both using the QT creator .pro files and directly from cmake. Both ways work for me.
I am using [this file](https://github.com/Artelnics/opennn/blob/master/examples/airfoil_self_noise/data/neural_network.xml) loading it as @GuillermoArtelnics said.

**Comment by Aleoli2** [comment]
I use the .pro and I was able to debug better the program. The error is produced in inputs_from_XML, in the line: 
`inputs_names[i] = input_element->GetText();`
My inputs don't have names, so this function return 0. I think that the error happens because 0 isn't a string and inputs_names is vector<string>.
In the function of tinyxml2 GetText() I have changed
`return 0;`
to
`return ""; `
and now it works.
